doc/api/esm.md

```diff
@@ -723,7 +723,12 @@ _isMain_ is **true** when resolving the Node.js application entry point.
 >       1. Return the result of **PACKAGE_MAIN_RESOLVE**(_packageURL_,
 >          _pjson_).
 >    1. Otherwise,
->       1. Return the URL resolution of _packageSubpath_ in _packageURL_.
+>       1. If _pjson_ is not **null* and _pjson.exports_ is *falsy*, then
```

Comment:
I think `null`/`undefined` is now the only special case. E.g. `exports: 42` is truthy but still has this behavior.